change in order to pass chk_fhan4 with pari-2.13

This patch is already upstream in giac > 1.6 so it will be
removed when we update giac.

--- a/src/pari.cc	2019-11-21 13:40:55.000000000 +0000
+++ b/src/pari.cc	2021-08-11 20:51:39.243502400 +0000
@@ -88,7 +88,7 @@
   static map<string,entree *> pari_function_table;
   static void do_giac_pari_init(long maxprime){
 #ifdef PARI_DYNAMIC_STACK
-    long pari_mem_size=512000;
+    long pari_mem_size=1024000;
 #else
     long pari_mem_size=64000000;
 #endif
